backend/src/templates/dto/get-template.dto.ts

```diff
@@ -11,3 +11,13 @@ export class GetTemplateResponseDto {
   body!: TemplateBlock[]
   paramsRequired!: string[]
 }
+
+export class GetTemplateMetaResponseDto {
```

Comment:
Sufficient for now, eventually we'll want to differentiate between what the user can do with each template (i.e. editor or issuer).

Reply:
Fix is up.